### ISSUE TITLE
Add Property Finder theme for Opportunity Zones

### ISSUE THREAD

**Issue by stevevance** [ChicagoCityscape/website#471]
Find and filter properties by zoning, class, and size

https://www.rejournals.com/unlocking-opportunity-zone-deals-in-illinois-20190108

"Off-market data as a necessity"

Until recently, commercial real estate data was historically guarded by big players in primary markets. Consequently, smaller firms in non-primary markets were required to budget a significant portion of time and resources toward the painstaking collection and analysis of publicly available data. People used to spend hours and days reviewing zoning regulations, transaction history and owner contact information, rendering the deal-making process slow and inefficient. But in the age of digital disruption, performing due diligence and research is no longer so laborious and pricey. The advent of off-market data aggregation tools has ushered in a new era of deal-making, saving stakeholders time and money.

The emergence of these data tools is very promising for the commercial real estate industry—especially for those in the business of finding Opportunity Zone investments, where demand is high and supply is becoming increasingly scarce. While scouring on-market listings can be an adequate place to begin searching for deals, this offers a limited view at best of a given market. So why obstruct your view and limit your possible investment options? The solution to this age-old supply and demand problem lies in leveraging off-market data sources.

Put simply, off-market properties are those that are not publicly listed for sale; a synonym could be “unlisted properties.” Since a limited number of properties are listed for sale at any given time, the breadth of off-market properties is dramatically larger.

**Comment by stevevance** [comment]
Built: https://www.chicagocityscape.com/propertyfinder.php?dataset=cook-county-opportunity-zones#15/41.8862/-87.6744
